NadavAI/utils.h

```diff
@@ -39,6 +39,10 @@ class Radian {
         return value_ * 180.0 / M_PI;
     }
 
+    Radian toRadian(float dergees) const {
```

Comment:
make this static, and call it `fromDegrees` (so: `Radian.fromDegrees(315.)`)

Reply:
(static because it doesn't access `this` and it doesn't make any sense for it to)